app/models/item.rb

```diff
@@ -9,4 +9,18 @@ class Item < ApplicationRecord
   validates :type, :price, :currency, :external_id, presence: true
   validates :price, numericality: { greater_than: 0 }
   validates :external_id, uniqueness: true
+
+  scope :by_brands, ->(brands) { brands.blank? ? all : where(brands.map { |brand| "brand ILIKE '%#{brand}%'" }.join(' OR ')) }
+  scope :by_model, ->(model) { model.blank? ? all : where('model ILIKE ?', "%#{model}%") }
+  scope :min_max_price, ->(min_price, max_price) { where('price > ? AND price < ?', min_price, max_price) }
```

Comment:
we should ignore condition if one of attribute is missing. So if `min_price` is absent we should use only `max_price`